modules/@ergonode/media/src/components/Buttons/LinkRelationButton.vue

```diff
@@ -5,19 +5,17 @@
 <template>
     <LinkButton
         :title="title"
-        @click.native="onClick" />
+        :class="[{ 'link-button--disabled': !hasLink }]"
+        @click.native="hasLink ? onClick(): null" />
```

Comment:
I would rather to move `hasLink` login into `onClick` and handle it there with providing event.preventDefault() and .stopPropagation() or even to css classes pointer-events: none when class --disabled is present